### ISSUE TITLE
Provide directional information on stops for VoiceOver users

### ISSUE THREAD

**Issue by aaronbrethorst** [OneBusAway/onebusaway-ios#530]
Feedback from a user who uses VoiceOver:

> i would like to see when looking at  bus stops for example near my house there are two bus stops on 76 ave. west, and 216th street sw.  these are community transit route 119.  one stops here on the east side of 76th goes to ash way p k, and the on on the south side of 76th goes to mountlake p k and ride.
> 
> these directions are not indicated when seeing the routes near me is selected.  this is across the board.
> i hope this can be addressed, please.  it is a time consuming step to do as a blind person using voice over.

I removed the cardinal directions from stops on the map recently. They should be restored for VoiceOver users specifically.

The user adds:

> You should have in that line
> For route 119 on 76th ave w. and 216th street southwest stop the stop going north on 76th should end with to ashway park and ride, then the details button.
 > 
> And the same route 119 going south on 76th should end with to mount lake terrace park and ride then details button.
 > 
> All the community transit stops should have their ending distination point before the details button.
 > 
> This is for vision inpaired users who cannot see the maps.
 > 
> Thanks for your time, and if you have any questions feel free to ask.
